src/app/pages/IdxPage/index.jsx

```diff
@@ -75,7 +73,7 @@ const IdxPage = ({
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <main role="main">
+      <main role="main" {...(service === 'ukrainian' && { lang })}>
```

Comment:
Could you please pull out this into a constant and add an explanatory comment about why this is added here? By looking at the code it isn't clear why  IDX pages with a service `ukrainian` should have a `lang` attribute on the `main` element.